spec/models/tag_spec.rb

```diff
@@ -77,6 +77,83 @@
         expect(@fandom_tag.taggings_count_cache).to eq 40 * ArchiveConfig.TAGGINGS_COUNT_CACHE_DIVISOR
       end
     end
+
+    context "write_redis_to_database" do
+      let(:tag) { create(:fandom) }
+      let!(:work) { create(:work, fandom_string: tag.name) }
+
+      before do
+        RedisJobSpawner.perform_now("TagCountUpdateJob")
+        tag.reload
+      end
+
+      it "does not write to the database when reading the count" do
+        tag.taggings_count
+        # Check if redis has flagged this tag for an update to the database.
+        expect(REDIS_GENERAL.sismember("tag_update", tag.id)).to eq false
```

Comment:
Probably worth adding a method like `expect_tag_flag_in_redis_to_be(tag, flag)` since these two lines are copy-pasted several times?

Reply:
Sure, I can do that when a larger review comes up.